packages/zmarkdown/zhtml.webpack.config.js

```diff
@@ -0,0 +1,24 @@
+const path = require('path')
+
+module.exports = {
+  entry: ['./plugins/client/zhtml'],
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    filename: 'zmarkdown-zhtml.js',
+    library: 'ZMarkdownZHTML',
+  },
+  module: {
+    rules: [
+      {
+        test: /\.jsx?$/,
+        exclude: /(node_modules|bower_components)/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            presets: ['env'],
```

Comment:
Will the `.babelrc` be used that way or should we add `target` in here as well?

Reply:
> cacheIdentifier: Default is a string composed by the babel-core's version, the babel-loader's version, the contents of .babelrc file if it exists, and the value of the environment variable BABEL_ENV with a fallback to the NODE_ENV environment variable. This can be set to a custom value to force cache busting if the identifier changes.
[source](https://webpack.js.org/loaders/babel-loader/#usage)

So I can just let:
```js
use: {
    loader: 'babel-loader',
},
```

Reply:
Same with a production build. Current target gives `92467B`, ie 6 target: `92467B`, no target `92467B`